### ISSUE TITLE
v1.2.0 Removes Set Text Color, Why?

### ISSUE THREAD

**Issue by searchy2** [GrenderG/Toasty#22]
Hi @GrenderG ,
This library is absolutely amazing and I am currently using it in 3 apps! Today when I upgraded from v1.1.5 to 1.2.0, the existing code broke because the set text color value was removed. Why was set text color removed? How is text color controlled now? Thank you for your great work!

**Comment by GrenderG** [comment]
Hello @searchy2, sorry for any inconveniences. You can change the text color and more parameters with this (check the `README.md`):

```java
Toasty.Config.getInstance()
    .setErrorColor(@ColorInt int errorColor) // optional
    .setInfoColor(@ColorInt int infoColor) // optional
    .setSuccessColor(@ColorInt int successColor) // optional
    .setWarningColor(@ColorInt int warningColor) // optional
    .setTextColor(@ColorInt int textColor) // optional
    .setToastTypeface(AssetManager assetManager, String fontPath) // optional
    .apply(); // required
```

**Comment by GrenderG** [comment]
Okay, now all the options are:
```java
Toasty.Config.getInstance()
    .setErrorColor(@ColorInt int errorColor) // optional
    .setInfoColor(@ColorInt int infoColor) // optional
    .setSuccessColor(@ColorInt int successColor) // optional
    .setWarningColor(@ColorInt int warningColor) // optional
    .setTextColor(@ColorInt int textColor) // optional
    .tintIcon(boolean tintIcon) // optional
    .setToastTypeface(AssetManager assetManager, String fontPath) // optional
    .apply(); // required
```
Remember to update to version `1.2.1`.

**Comment by searchy2** [comment]
Thanks for the care you put into the library and continually making it better. I think I understand how the changes work. I have one question that I am still confused about that I hope you can clear up.

Is there a method to set the color of a custom toast? I see that you can set Info, Error, and Warning toast colors. How do you set it for the custom toast? Thanks!

**Comment by GrenderG** [comment]
Thanks for your words. You can still change a custom Toast color [the same way as before](https://github.com/GrenderG/Toasty/blob/master/toasty/src/main/java/es/dmoral/toasty/Toasty.java#L163-L165). The `custom()` method accepts a ` @ColorInt int tintColor` parameter which corresponds to the color that the Toast will have.

**Comment by searchy2** [comment]
Oh! I thought the tint color was used to specify the color for the text and icon. I did not realize that it was for the background color. Will try to upgrade now, hopefully everything goes smoothly!

**Comment by GrenderG** [comment]
Hope it goes smoothly. :smile: By the way, I've released version `1.2.3`. The only noticeable changes take place on this method:
```java
.setToastTypeface(@NonNull Typeface typeface) // optional
```
Now it accepts a Typeface object.

**Comment by searchy2** [comment]
Everything works smoothly!
